### PR TITLE
Announce page change using aria-live=assertive

### PR DESCRIPTION
**Pull request by onmax**
#18

### CODE REVIEW

File: test/main.spec.ts [hunk 1]
```diff
@@ -67,37 +57,43 @@ test('meta tags are added and removed', async ({ page }) => {
...
-});
\ No newline at end of file
+  const announcerHome = page.locator('#flamethrower-announcer');
+  await expect(announcerHome).toHaveText('Flamethrower');
+
+  const about = page.locator('#about');
+  await about.click();
+
+  const announcerAbout = page.locator('#flamethrower-announcer');
+  await expect(announcerAbout).toHaveText('About');
```
Comment (AriPerkkio) [inline test/main.spec.ts:98]: FYI, there is a tool for validating these announcements. https://www.npmjs.com/package/extend-to-be-announced
Though I'm not sure whether it integrates with playwright nicely.

Comment (onmax) [reply test/main.spec.ts:98]: Nice, I didn't know about this one. Although, not sure if it is a good idea add a new dev dependency only for this test case 🤔.

Another way of optimizing the test that I was thinking is:

I could use the same variable `const announcer = page.locator('#flamethrower-announcer');` for `home` page and `about` page as technically is the same node, so maybe it keeps the reference, but I didn't try it and I don't know if it works

**Comment by codediodeio** [comment]
This looks good, but we've got some conflicts to resolve now.

**Comment by onmax** [comment]
I am a little bit lost, and first time I am doing this. Not sure what `git` commands to use:

Maybe this:
```bash
git config pull.rebase false
git pull upstream pull
# Merge conflicts
git commit -m "Merge blah blah blah"
git push
```

Or what do you guys recommend?

**Pull request closed without merging**